### ISSUE TITLE
Error in media/photo.py

### ISSUE THREAD

**Issue by zserg** [jmathai/elodie#46]
It seems like error in photo.py line 82:
````   
 if(key == 'latitude'):
````
should be:
```` 
if(type == 'latitude'):
````

**Comment by jmathai** [comment]
`key` is set based on the value of `type` earlier on [line 66](https://github.com/jmathai/elodie/blob/master/elodie/media/photo.py#L66).

```
key = self.exif_map['longitude'] if type == 'longitude' else self.exif_map['latitude']
```

**Comment by zserg** [comment]
The image 'with-location-inv.jpg'  have еру сoordinates inverted relative to 'with-location.jpg'. Tests added was failed. zserg@ea04b77dc5db74be77633f04c62865f56d245a6e

The following fix made test passed: zserg@95923e7942a63fd00d6c302f6b6539376f1206f7

**Comment by jmathai** [comment]
@zserg You're correct. Thanks for commenting despite me closing the issue. Do you mind opening a PR? I would suggest this additional change to make the code more readable. We should use `type` when we can and `key` for when we need the `exivpy2` attribute name.

```
--- a/elodie/media/photo.py
+++ b/elodie/media/photo.py
@@ -72,14 +72,14 @@ class Photo(Media):
         try:
             # this is a hack to get the proper direction by negating the values for S and W
             latdir = 1
-            if(key == self.exif_map['latitude'] and str(exif[self.exif_map['latitude_ref']].value) == 'S'):
+            if(type == 'latitude' and str(exif[self.exif_map['latitude_ref']].value) == 'S'):
                 latdir = -1
             londir = 1
-            if(key == self.exif_map['longitude'] and str(exif[self.exif_map['longitude_ref']].value) == 'W'):
+            if(type == 'longitude' and str(exif[self.exif_map['longitude_ref']].value) == 'W'):
                 londir = -1
